src/cli/test_command.zig

```diff
@@ -327,7 +327,7 @@ pub const TestCommand = struct {
         js_ast.Expr.Data.Store.create(default_allocator);
         js_ast.Stmt.Data.Store.create(default_allocator);
         var vm = try JSC.VirtualMachine.init(ctx.allocator, ctx.args, null, ctx.log, env_loader);
-        vm.argv = ctx.positionals;
+        vm.argv = ctx.passthrough;
```

Comment:
we should also test that this one works via running `bun wiptest`

Reply:
`bun-debug wiptest process-args.test.js` is okay, but `bun-debug wiptest` isn't happy with the json parsing:
```
const t1 = JSON.parse(await new Response(s1).text());
                   ^
SyntaxError: JSON Parse error: Unexpected EOF
      at /Users/dylan/code/bun/test/bun.js/process-args.test.js:12:15
```

Reply:
nevermind, it is working with `bun-debug wiptest`. I ran it from the wrong directory and there are other error that got in the way